src/services/NodeService.ts

```diff
@@ -34,37 +49,65 @@ export class NodeService {
+        const profileNode = this.loadNodes(profile);
+        return _.uniqBy(profileNode, 'url').filter((n) => n.networkType === profile.networkType);
     }
 
-    public getNodes(profile: ProfileModel, repositoryFactory: RepositoryFactory, repositoryFactoryUrl: string): Observable<NodeModel[]> {
-        const storedNodes = this.getKnowNodesOnly(profile);
+    public async getNodes(
+        profile: ProfileModel,
+        repositoryFactory: RepositoryFactory,
+        repositoryFactoryUrl: string,
+        isOffline?: boolean,
+    ): Promise<NodeModel[]> {
+        const storedNodes = this.getKnownNodesOnly(profile);
+        const statisticsNodes = !isOffline ? await this.getNodesFromStatisticService(profile.networkType) : [];
         const nodeRepository = repositoryFactory.createNodeRepository();
-
-        return nodeRepository.getNodeInfo().pipe(
-            map((dto: NodeInfo) =>
-                this.createNodeModel(
-                    repositoryFactoryUrl,
-                    profile.networkType,
-                    dto.friendlyName,
-                    undefined,
-                    dto.publicKey,
-                    dto.nodePublicKey,
+        return nodeRepository
+            .getNodeInfo()
+            .pipe(
+                map((dto: NodeInfo) =>
+                    this.createNodeModel(
+                        repositoryFactoryUrl,
+                        profile.networkType,
+                        dto.friendlyName,
+                        undefined,
+                        dto.publicKey,
+                        dto.nodePublicKey,
+                    ),
                 ),
-            ),
-            ObservableHelpers.defaultLast(this.createNodeModel(repositoryFactoryUrl, profile.networkType)),
-            map((currentNode) => _.uniqBy([currentNode, ...storedNodes], 'url')),
-            tap((p) => this.saveNodes(profile, p)),
-        );
+                ObservableHelpers.defaultLast(this.createNodeModel(repositoryFactoryUrl, profile.networkType)),
+                map((currentNode) => _.uniqBy([currentNode, ...storedNodes], 'url')),
+                tap((p) => this.saveNodes(profile, p)),
+            )
+            .toPromise()
+            .then((val) => (statisticsNodes && statisticsNodes.length ? _.uniqBy(val.concat([...statisticsNodes]), 'url') : val));
     }
 
-    private loadStaticNodes(networkType: NetworkType): NodeModel[] {
-        return networkConfig[networkType].nodes.map((n) => {
-            return this.createNodeModel(n.url, networkType, n.friendlyName, true);
-        });
+    public async getNodesFromStatisticService(networkType: NetworkType, isOffline?: boolean): Promise<NodeModel[]> {
+        const nodeSearchCriteria = {
+            nodeFilter: 'suggested',
+            limit: 30,
+        };
+        if (!isOffline && navigator.onLine) {
+            const data = await requestNodes(networkType, nodeSearchCriteria);
```

Comment:
Probably it'd better to use the openapi-client (symbol-statistics-service-typescript-fetch-client) instead.

Here is how it's done in bootstrap: https://github.com/symbol/symbol-bootstrap/blob/c846846245e02d8886fac5985374da0a15b05a83/src/service/RemoteNodeService.ts#L136

Reply:
@yilmazbahadir @cryptoBeliever  Can we open a new issue for this as an enhancement? and maybe we should consider that into mobile wallet too.

Reply:
I'm fine with this

Reply:
@bassemmagdy created and assigned to you https://github.com/symbol/desktop-wallet/issues/1757